service/web_api.py

```diff
@@ -195,6 +195,33 @@ def check_model_exist():
     return jsonify({"code": 0, "message": "success", "exists": result_list})
 
 
+@app.route("/api/checkURLExists", methods=["POST"])
+def check_url_exists():
```

Comment:
maybe a more exact/specific title: check_if_huggingface_repo_exists? 
I am not too familiar with the hugging face terminology. repo_id was used all over the code base beforehand, maybe we stick with this for now